src/main/java/com/corndel/cashregister/Level4.java

```diff
@@ -10,6 +12,18 @@ public class Level4 {
    */
   public static boolean canMakeAmount(int target, List<Item> drawer) {
     // TODO
-    return false;
+
+    List<Item> sortedDrawer = new ArrayList<>(drawer);
+    sortedDrawer.sort((a, b) -> b.value - a.value);
```

Comment:
Good work researching this method and kudos for implementing the lambda method you are passing in.

Funnily enough, this lambda is conforming to the Comparator interface, which is similar to the Comparable interface we looked at today.

Side note: I do not think you need to sort the list first, will this pass the tests if you comment it out?